auth-api/src/auth_api/models/entity.py

```diff
@@ -38,6 +38,7 @@ class Entity(BaseModel):  # pylint: disable=too-few-public-methods
     business_number = Column('business_number', String(100), nullable=True)
     name = Column('name', String(250), nullable=True)
     corp_type_code = Column(String(10), ForeignKey('corp_type.code'), nullable=False)
+    folio_number = Column('folio_number', String(50), nullable=True, index=True)
```

Comment:
just making sure , does this filed need index=true

Reply:
I just copied over from pay-api.